docs/content/learn/tutorials/metadata/_index.md

```diff
@@ -0,0 +1,783 @@
+- [`document-ids`](https://pages.nist.gov/OSCAL/documentation/schema/catalog-layer/catalog/json-schema/##gen_metadata-oscal-metadata-document-id_ref-h3) (Optionally Many) - A unique ID that identifies a document, and ties all separate instances of the document into one document series. The `scheme` field provides a URI to identify the scheme used to generate the ID. See  for more information and usage guidance.
+- [`props`](https://pages.nist.gov/OSCAL/documentation/schema/catalog-layer/catalog/json-schema/##gen_metadata-property_ref-h3) (Optionally Many) - Represents some arbitrary property of the document. This flexible element provides an extension point for adding additional metadata. See TODO for more information and usage guidance.
+- [`links`](https://pages.nist.gov/OSCAL/documentation/schema/catalog-layer/catalog/json-schema/##gen_metadata-link_ref-h3) (Optionally Many) - Provides a resolvable URI (the `href` field) to some resource. The purpose of the link is given with the `rel` field, and it's media type through the `media-type` field. See [here](https://pages.nist.gov/OSCAL/tutorials/metadata/#understanding-and-using-document-id) for more information and usage guidance.
+- [`roles`](https://pages.nist.gov/OSCAL/documentation/schema/catalog-layer/catalog/json-schema/##gen_metadata-role_ref-h3) (Optionally Many) - Defines a function assumed or expected to be assumed by a party in a specific situation.
+- [`locations`](https://pages.nist.gov/OSCAL/documentation/schema/catalog-layer/catalog/json-schema/##gen_metadata-location_ref-h3) (Optionally Many) - A location, with associated metadata that can be referenced.
+- [`parties`](https://pages.nist.gov/OSCAL/documentation/schema/catalog-layer/catalog/json-schema/##gen_metadata-party_ref-h3) (Optionally Many) - A responsible entity which is either a person or an organization.
+- [`responsible-parties`](https://pages.nist.gov/OSCAL/documentation/schema/catalog-layer/catalog/json-schema/##gen_metadata-responsible-party_ref-h3) (Optionally Many) - A reference to a set of organizations or persons that have responsibility for performing a referenced role in the context of the containing object.
+- [`remarks`](https://pages.nist.gov/OSCAL/documentation/schema/catalog-layer/catalog/json-schema/##gen_metadata-remarks_ref-h3) (Optional) - Markup formatted plaintext with notes for human readers of the content.
+
+{{% /tab %}}
+{{% /tabs %}}
+## Creating a Metadata Section
+
+All OSCAL documents require a Metadata section with a number of basic elements. This tutorial will cover creating this section and populating the required elements. We will also cover several important optional elements, and how they can be used to achieve specialized use cases.
+
+Lets start with a nominal example OSCAL document to explain how you would create Metadata. 
+
+This OSCAL document is a Catalog, called "Example OSCAL Catalog". It was created today (1/1/2021) by me, Example Person, employee of Example Company.
```

Comment:
I feel that both examples are effectively the same already (Example OSCAL Catalog and Sample Security Catalog). Is there value in making sure that they are the same?

Reply:
I was hoping we can provide some level of gradual progression and continuity among all tutorials (at least where ever is possible)

Reply:
While I think this might be helpful, some decoupling may be needed. Maintaining too much connectivity between examples in the tutorials is going to make these hard to maintain as we make changes.